packages/report-flaky-tests/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "@wordpress/report-flaky-tests",
+	"version": "0.0.0",
```

Comment:
I'm never 100% about this but I think we want this to be `1.0.0-prerelease` so that the first publish (once `"private": true` is removed) is `1.0.0`.

Reply:
I don't have a strong opinion about this, but I think that `1.0.0` is considered the first _stable_ release in semver, and maybe we should start with `0.x.x` for experimental/private package?

Reply:
Yeah fair enough. Not sure if the project has a preference. Maybe @gziolo knows.

Reply:
I think it doesn't matter here in case of package for development purposes.